File: test-email.js
const admin = require('firebase-admin');
const serviceAccount = require('./functions/serviceAccountKey.json');

admin.initializeApp({
  credential: admin.credential.cert(serviceAccount),
  databaseURL: "https://balkantvpanel.firebaseio.com"
});

const db = admin.firestore();

async function sendTestEmail() {
  try {
    const result = await db.collection('mail').add({
      to: 'balkan-iptv@outlook.com',
      message: {
        subject: 'Test Mail - Balkan TV Panel',
        text: 'Bu bir test emailidir. Email sistemi çalışıyor mu kontrol etmek için gönderildi.'
      },
      createdAt: new Date()
    });

    console.log('✅ Test belgesi başarıyla oluşturuldu!');
    console.log('Document ID:', result.id);
    console.log('\n📧 Email adresine gitmesini bekleyin (2-5 dakika)');
    console.log('Email adresi: balkan-iptv@outlook.com');
    
    process.exit(0);
  } catch (error) {
    console.error('❌ Hata:', error);
    process.exit(1);
  }
}

sendTestEmail();

File: test_email.py
#!/usr/bin/env python3
"""
Firebase Firestore'a test email belgesi eklemek için script
"""

import subprocess
import json
import sys
from datetime import datetime

def add_test_email():
    """Firestore'a test email belgesi ekler"""
    
    # Test belgesi
    test_doc = {
        "to": "mersinakpinar@outlook.com",
        "message": {
            "subject": "Test Mail - Balkan TV Panel",
            "text": "Bu bir test emailidir. Email sistemi çalışıyor mu kontrol etmek için gönderildi."
        },
        "createdAt": datetime.now().isoformat()
    }
    
    # firebase CLI komutu - Firestore'a doğrudan ekleme
    cmd = [
        "firebase",
        "firestore",
        "databases",
        "describe"
    ]
    
    try:
        print("📧 Test email belgesi oluşturuluyor...")
        print(f"   Alıcı: mersinakpinar@outlook.com")
        print(f"   Konu: Test Mail - Balkan TV Panel")
        print("\n⏳ Lütfen Firebase Console'u açın ve şu adımları takip edin:")
        print("\n1. https://console.firebase.google.com/project/balkantvpanel")
        print("2. Firestore Database > 'mail' koleksiyonuna gidip")
        print("3. 'Belge Ekle' tıklayıp bu JSON'u yapıştırın:\n")
        
        print(json.dumps(test_doc, indent=2, ensure_ascii=False))
        
        print("\n4. Kaydet tıklayın")
        print("5. 2-5 dakika bekleyin")
        print("6. mersinakpinar@outlook.com mail kutusunu kontrol edin\n")
        
    except Exception as e:
        print(f"❌ Hata: {e}")
        sys.exit(1)

if __name__ == "__main__":
    add_test_email()
